content/spin/v2/install.md

```diff
@@ -286,10 +286,74 @@ Then install plugins by name.
 
 Python:
 
+> Historic: You may have see a `py2wasm` plugin in your travels. Please note, `py2wasm` (an experiment to build a Spin Python SDK using CPython, Wizer, and PyO3) has since been replaced by `componentize-py`. 
```

Comment:
```suggestion
> Historic: You may have seen a `py2wasm` plugin in your travels. Please note, `py2wasm` has since been replaced by `componentize-py`. 
```